controllers/pkg/reconcilers/token/README.md

```diff
@@ -0,0 +1,49 @@
+# token controller
+
+The token controller is a k8s controller acting on token.infra.nephio.org and handles the lifecycle of the token in gitea. It also adds a corresponding secret in k8s within the namespace the token was applied.
+
+The token is immutable, so if you want to change the token it has to be deleted/created
+
+## implementation
+
+The implementation assumes the repo-controller runs in the same cluster as the gitea server. Based on the environment variables we help the controller to connect to the gitea server.
```

Comment:
in the manifest packages there is a separate yaml for this. I split token and repo since token is independent on repo and we need to deal with different way's configsync/porch handles them, so this Is why I split and in the repo package we also have tokens we will apply to the cluster.